research/flamby/fed_heart_disease/fenda/server.py

```diff
@@ -20,7 +20,7 @@
 )
 
 
-def main(config: Dict[str, Any], server_address: str) -> None:
+def main(config: Dict[str, Any], server_address: str, run_name: str) -> None:
```

Comment:
It doesn't look like this run name is used anywhere? Other servers use run_name to save a server-side checkpoint, but FENDA doesn't have "server-side" models. So it hasn't needed this value in the past.